icu4c/source/data/brkitr/rules/line_loose_phrase_cj.txt

```diff
@@ -127,7 +126,8 @@ $dictionary     = [$ComplexContext $dictionaryCJK];
 #                               XX  (Unknown, unassigned)
 #                         as $AL  (Alphabetic)
 #
-$ALPlus = [$AL $AI $SG $XX $EAFWH [$dictionary-[[:Mn:][:Mc:]]]];
+$EAW = [[:Lu:][:Ll:][\uff10-\uff19]];
```

Comment:
Thanks for reducing the additions. However, the `[:Lu:][:Ll:]` *look* large, and it's hard to see what really gets added.
Let's be specific, for example:
```suggestion
# Let fullwidth-ASCII digits and letters be part of words.
$FW_alphanum = [０-９Ａ-Ｚａ-ｚ];
```